abci/types/result.go

```diff
@@ -41,6 +41,26 @@ func (r ResponseQuery) IsErr() bool {
 	return r.Code != CodeTypeOK
 }
 
+// IsUnknown returns true if Code is Unknown
+func (r ResponseProcessProposal) IsUnknown() bool {
+	return r.Result == ResponseProcessProposal_UNKNOWN
+}
+
+// IsOK returns true if Code is OK
+func (r ResponseProcessProposal) IsOK() bool {
+	return r.Result == ResponseProcessProposal_ACCEPT
+}
+
+// IsErr returns true if Code is something other than OK.
+func (r ResponseProcessProposal) IsErr() bool {
+	return r.Result != ResponseProcessProposal_ACCEPT
+}
```

Comment:
This includes value ResponseProcessProposal_REJECT, which I find misleading.
I checked the source code and this method is used nowhere in the repo, so it's probably best if you just remove it.